src/hooks/usePlaceOrder.ts

```diff
@@ -47,23 +47,41 @@ export const usePlaceOrder = (): Result => {
           exchangeApi.getCurrentBatchId(),
         ])
 
-        const validUntil = batchId + DEFAULT_ORDER_DURATION
+        if (sellTokenId !== 0 || buyTokenId !== 0) {
```

Comment:
As we discussed offline, this code uses `idToAddressBiMap` lib which requires the id to exist https://github.com/gnosis/solidity-data-structures/blob/9d0c8c05b85ba6c28fa5717ca1ac9bdb60ee117d/contracts/libraries/IdToAddressBiMap.sol#L19

Thus, it should throw an error in that case, rather than return 0.

We need to check the contract you are testing against to figure out what tokes are registered.

Reply:
Yep, but for some reason we are getting 0s there. We need to investigate a bit more why